sentinel-adapter/sentinel-spring-webmvc-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/SentinelSpringMvcIntegrationTest.java

```diff
@@ -107,13 +109,16 @@ public void testTotalInterceptor() throws Exception {
     @Test
     public void testRuntimeException() throws Exception {
         String url = "/runtimeException";
-        configureExceptionRulesFor(url, 3, null);
-        int repeat = 3;
+
+        configureExceptionRulesFor(url, 4.2, null);
```

Comment:
It is recommended to add a separate test method while keeping the original flowRule test cases.

Reply:
> It is recommended to add a separate test method while keeping the original flowRule test cases.

First of all, i have raised this pr for a long time ,thanks for  your review ~ In my first view, I prefer adding a separate test.However ,when I ran test method `testRuntimeException` ,I was finding some error. `FlowRule` is not match with
exception.Maybe you can read source code about `DegradeRule`
```
    private void configureExceptionRulesFor(String resource, int count, String limitApp) {
        FlowRule rule = new FlowRule()
                .setCount(count)
                //DEGRADE_GRADE_EXCEPTION_RATIO with FlowRule? I guess the coder wrote this wanted to set degrade rule
                .setGrade(RuleConstant.DEGRADE_GRADE_EXCEPTION_RATIO);
        rule.setResource(resource);
        if (StringUtil.isNotBlank(limitApp)) {
            rule.setLimitApp(limitApp);
        }
        FlowRuleManager.loadRules(Collections.singletonList(rule));
    }


    /**
    * FlowRule
     * The threshold type of flow control (0: thread count, 1: QPS).
     */
    private int grade = RuleConstant.FLOW_GRADE_QPS;
```

Reply:
I think the original scenario only test the rate limiting situation. The name "testRuntimeException" is indeed not appropriate for describing it. However, it is still necessary to keep this test case. Could you please modify it? Thank you for your assistance.

Reply:
After read origin code again,I think the function test exception situation and reason blow
- Test funtion names `testRuntimeException`  and `com.alibaba.csp.sentinel.adapter.spring.webmvc.controller.TestController#runtimeException` do throw a runningtime exception
- LoadRule function'name is `configureExceptionRulesFor`  not  `configureRulesFor`
- There is no degrade rule test,but flow test. 
Need we modify this test?  Is it still necessary to keep this test case?How about your advice?

Reply:
Yes, we need modify this test, my advice is that you can create a new unit test for degrade rule within the current PR. this test is unrelated to the current functionality, and you can create a new PR to fix the flow rule test.

Reply:
Done~ Could you have a look?